server/routes/api/subscriptions/subscriptions.ts

```diff
@@ -164,12 +165,13 @@ router.get(
 
     const [subscription, user] = await Promise.all([
       Subscription.findOne({
-        where: {
-          userId,
-          documentId,
-        },
+        where: subscriptionId
+          ? { id: subscriptionId }
+          : {
+              userId,
+              documentId,
+            },
         lock: Transaction.LOCK.UPDATE,
-        rejectOnEmpty: true,
```

Comment:
Removed on purpose, previously this would just show a 404 error if you were already unsubscribed – not a great UX.